app/Http/Transformers/Legacy/Two/PostTransformer.php

```diff
@@ -67,6 +67,10 @@ public function transform(Post $post)
      */
     public function includeSignup(Post $post)
     {
+        if (! $post->signup) {
+            return;
```

Comment:
I feel like there should be a more elegant response to indicate that there is no signup included with the post because it has been deleted but when I trie dto do a custom message in a simple array, I'd get this message: 
```
Exception: Invalid return value from League\Fractal\TransformerAbstract::includeSignup(). Expected League\Fractal\Resource\ResourceAbstract, received array. in file /home/vagrant/DoSomething.org/rogue/vendor/league/fractal/src/TransformerAbstract.php on line 185
```

Reply:
I think returning `null` here is great!